backend/app/auth/auth_exception.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+from app.models.types import NonBlankString
+
+
+class AuthException(ABC, BaseException):
```

Comment:
Should probably switch the order of the inheritance here.

Reply:
Yes, order of inheritance still matters, but `AuthException` is no longer an ABC so it doesn't matter here anymore.